roslibrust/src/rosbridge/integration_tests.rs

```diff
@@ -135,9 +135,13 @@ mod integration_tests {
         // Intentionally a port where there won't be a server at
         let opts = ClientHandleOptions::new("ws://localhost:9091").timeout(TIMEOUT);
         assert!(ClientHandle::new_with_options(opts).await.is_err());
+
+        // This case sometimes actually passed in CI and caused test to fail
```

Comment:
I think if we want to keep this case it could be better served by directly spawning with `tokio::task::spawn` but not awaiting it, yielding back to tokio manually (`tokio::task::yield_now()`), then manually calling poll on the handle returned from spawning. I think this allows forcing the timeout.

Reply:
Agreed, but yeah I don't think we need it re-enabled at the moment.